### ISSUE TITLE
Segmentation fault

### ISSUE THREAD

**Issue by jptaranto** [JohnAlbin/chroma#20]
Here is a really strange one.

gulp-sass 2.3.1
node-sass 3.6.0
chroma 1.2.3
node 5.11.0

Sass is throwing a Segmentation Fault when the chroma-kss-markup() function is used.

I tracked it down to line 77 to 82:

```sass
        $markup: $markup
          + '<div class="#{$color-class}">'
          +   '<span class="#{$swatch-class}" style="background-color: #{$value}"></span>'
          +   '<code class="#{$variable-class}">#{$_chroma-spelling}(#{$color})</code>'
          +   ' <span class="#{$alt-text-class}">uses the #{$_chroma-spelling}:</span> '
          +   '<span class="#{$value-class}"><code>#{$value}</code>';
```

It seems node-sass doesn't like so many string concats being used in one line. Changing to the below allowed it to build normally.

```sass
        $markup: $markup
          + '<div class="#{$color-class}">'
          +   '<span class="#{$swatch-class}" style="background-color: #{$value}"></span><code class="#{$variable-class}">#{$_chroma-spelling}(#{$color})</code> <span class="#{$alt-text-class}">uses the #{$_chroma-spelling}:</span> <span class="#{$value-class}"><code>#{$value}</code>';
```

So almost certainly a strange bug in node-sass but an easy fix here.

What do you think John?

**Comment by JohnAlbin** [comment]
Node-sass 3.5 was breaking Zen 7.x-6.x's Sass, so I had to pin its package.json to use node-sass 3.4.

I haven't tried node-sass 3.6 yet.

**Comment by JohnAlbin** [comment]
I'm seeing this in the Travis tests now. Thanks, node-sass! https://travis-ci.org/JohnAlbin/chroma/builds/126840703

**Comment by jptaranto** [comment]
Fast but prone to errors!

Cheers

**Comment by JohnAlbin** [comment]
Thanks, @jptaranto! I've released 1.2.4 with the fix.
